.ycm_extra_conf.py

```diff
@@ -86,6 +86,7 @@ def pio_pkg_dir(pkg):
     "-fno-exceptions",
     "-Wall",
     "-Werror=conversion",
+    "-Wno-sign-conversion",
```

Comment:
Should we have this in platformio.ini too?

Reply:
I think clang enables it as a result of -Wconversion but gcc doesn't for C++ code

https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html
> -Wsign-conversion
> Warn for implicit conversions that may change the sign of an integer value, like assigning a signed integer expression to an unsigned integer variable. An explicit cast silences the warning. In C, this option is enabled also by -Wconversion.

Will do in a separate PR.